src/install.c

```diff
@@ -1099,6 +1099,66 @@ static RaucSlot* get_boot_mark_slot(const GPtrArray *install_plans)
 	return bootslot;
 }
 
+#define MESSAGE_ID_INSTALLATION_STARTED   "b05410e8-a933-4538-9cd0-61aab1e9516d"
+#define MESSAGE_ID_INSTALLATION_SUCCEEDED "0163db54-68ac-4237-b090-d28490c301ed"
+#define MESSAGE_ID_INSTALLATION_FAILED    "c48141f7-fd49-443a-afff-862b4809168f"
+#define MESSAGE_ID_INSTALLATION_REJECTED  "60bea7e4-fea5-49cc-ad68-af457308b13a"
+
+static void log_event_installation_started(RaucInstallArgs *args)
+{
+	g_log_structured(R_EVENT_LOG_DOMAIN, G_LOG_LEVEL_MESSAGE,
+			"RAUC_EVENT_TYPE", "install",
+			"MESSAGE_ID", MESSAGE_ID_INSTALLATION_STARTED,
+			"TRANSACTION_ID", args->transaction,
+			"MESSAGE", "Installation %.8s started", args->transaction
```

Comment:
Why do you use `%.8s` instead of `%s`?

Reply:
because we have the full transaction ID in `TRANSACTION_ID` anyway and I wanted to keep the text output readable. Even Git does not use full SHA356 in log messages but only shows 8 characters.

Reply:
```suggestion
			"MESSAGE", "Installation %.8s started", args->transaction, # truncate ID for readability
```

Reply:
Fine for me with C comment :wink: